woocommerce/woocommerce-android
# [Mobile Payments] Enable NFC action to the "NFC disabled" error state

Closes: #9610 


### Description

The PR adds "enable NFC" button to the NFC disabled error state. The click on the button opens android NFC settings

### Testing instructions

* Use release build
* Disable NFC
* Payments ->  Collect -> TTP
* Notice the new button. Click on it
* Enabled NFC and come back
* Notice that TTP works now

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/bee27806-42b1-42f0-802e-e6045f50ef67
